python/app/g2panel

```diff
@@ -418,15 +429,25 @@ class MainWindow(QtWidgets.QMainWindow):
     def triggerMode(self, selected):
         if selected:
             self.det.timing = timingMode.TRIGGER_EXPOSURE
+        
 
     #Safely powering off the detectors
     def powerOffDetector(self):
-        self.det.highvoltage = 0
-        self.highVoltageSpinBox.setValue(self.det.highvoltage)
-        time.sleep(1)
-        #TODO yet to implement with the initialization of the chip
-        #self.det.powerchip = False
-
+        #Asking user if they really want to power off the detectors
+        powerDlg = QtWidgets.QMessageBox()
+        powerDlg.setWindowTitle("Power Off the Detecor!!!")
+        powerDlg.setText("Do you really want to power off the Detectors?")
```

Comment:
Most likely it will be a single detector so change to: Do you really want to power off the detector?

Reply:
It must be typo but I made the changes